test/command.js

```diff
@@ -171,6 +176,41 @@ test('$ handles invalid escape sequence', async t => {
 	t.is(stdout, '\\u');
 });
 
+test('$ can concatenate at the end of tokens', async t => {
+	const {stdout} = await $`echo.js foo${'bar'}`;
+	t.is(stdout, 'foobar');
+});
+
+test('$ does not concatenate at the end of tokens with a space', async t => {
+	const {stdout} = await $`echo.js foo ${'bar'}`;
+	t.is(stdout, 'foo\nbar');
+});
+
+test('$ can concatenate at the end of tokens followed by an array', async t => {
+	const {stdout} = await $`echo.js foo${['bar', 'foo']}`;
+	t.is(stdout, 'foobar\nfoo');
+});
+
+test('$ can concatenate at the start of tokens', async t => {
+	const {stdout} = await $`echo.js ${'foo'}bar`;
+	t.is(stdout, 'foobar');
+});
+
+test('$ does not concatenate at the start of tokens with a space', async t => {
+	const {stdout} = await $`echo.js ${'foo'} bar`;
+	t.is(stdout, 'foo\nbar');
+});
+
+test('$ can concatenate at the start of tokens followed by an array', async t => {
+	const {stdout} = await $`echo.js ${['foo', 'bar']}foo`;
+	t.is(stdout, 'foo\nbarfoo');
+});
```

Comment:
Tested locally, but this might be another good one to include:

```js
test('$ can concatenate at the start and end of tokens surrounding an array', async t => {
	const {stdout} = await $`echo.js foo${['bar', 'baz']}foo`;
	t.is(stdout, 'foobar\nbazfoo');
});
```

Reply:
Good idea! Added in 366e6fcfe74bab8f5b074c9c0b8c70d9d974f88e